### ISSUE TITLE
Make sure `Run QR code action` process can be executed from everywhere

### ISSUE THREAD

**Issue by teosarca** [metasfresh/metasfresh-webui-frontend#2015]
### Is this a bug or feature request?
Bug

required for https://github.com/metasfresh/metasfresh/issues/4577

#### Which are the steps to reproduce?
* open a particular order, e.g. https://w101.metasfresh.com:8443/window/143/1001238
* from main menu, try to run `Run QR code action` process
* write something in that Barcode field and press enter => nothing happens, frontend is not calling the `/start` endpoint

If you would try to run the process but while u are on Dashboard (i.e. https://w101.metasfresh.com:8443) everything would work.

### What is the expected or desired behavior?
process shall be executed no matter where the user is

**Comment by siemiatj** [comment]
It wasn't lined up in the requirements anywhere, what process is it supposed to start ? You're referring to this url : `${config.API_URL}/process/${processType}/${pinstanceId}/start` ? Where will I get the pinstanceId and processType from ?

**Comment by teosarca** [comment]
yes, but in that case, frontend simply does nothing when i press enter
